src/core/folder.h

```diff
@@ -168,7 +168,7 @@ private Q_SLOTS:
     bool wants_incremental;
     bool stop_emission; /* don't set it 1 bit to not lock other bits */
 
-    std::unordered_map<const char*, std::shared_ptr<const FileInfo>, CStrHash, CStrEqual> files_;
+    std::unordered_map<const std::string, std::shared_ptr<const FileInfo>, std::hash<std::string>> files_;
```

Comment:
`std::hash<Key>` is for `std::string`, not for `const std::string`. Compilation would fail without adding it.